doc/user/content/sql/drop-table.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "DROP TABLE"
+description: "`DROP TABLE` removes a table from your Materialize instance."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`DROP TABLE` removes a table from your Materialize instance.
+
+## Conceptual framework
+
+[Tables](../create-table) store non-streaming data that is inserted via [INSERT](../insert)
+statements. `DROP TABLE` removes tables from your Materialize instance.
+
+## Syntax
+
+{{< diagram "drop-table.svg" >}}
+
+Field | Use
+------|-----
+**IF EXISTS** | Do not return an error if the named table does not exist.
+_table&lowbar;name_ | The name of the table to drop.
+
+## Details
+
+### Restrictions
```

Comment:
These restrictions don't seem very applicable for dropping a table.

Reply:
Oops yes, good point!